internal/config/config.go

```diff
@@ -14,7 +14,7 @@ const Filename = ".ldcli-config.yml"
 // ConfigFile represents the data stored in the config file.
 type ConfigFile struct {
 	AccessToken     string `json:"access-token,omitempty" yaml:"access-token,omitempty"`
-	AnalyticsOptOut bool   `json:"analytics-opt-out,omitempty" yaml:"analytics-opt-out,omitempty"`
+	AnalyticsOptOut *bool  `json:"analytics-opt-out,omitempty" yaml:"analytics-opt-out,omitempty"`
```

Comment:
The `omitempty` would kick in without this, so a `false` value would not get displayed with `config --list`.

Reply:
ooops thanks for catching

Reply:
We _should_ have tests that write the config to an in-memory file system and read it back to catch stuff like this, but I didn't spend the effort on it.